test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsEndToEndTestsBase.cs

```diff
@@ -287,6 +287,7 @@ public async Task Validate_HostLogs()
             ValidateTrace(traces[idx++], "Host lock lease acquired by instance ID", ScriptConstants.LogCategoryHostGeneral);
             ValidateTrace(traces[idx++], "Host started (", LogCategories.Startup);
             ValidateTrace(traces[idx++], "Initializing Host", LogCategories.Startup);
+            ValidateTrace(traces[idx++], "Initializing Warmup Extension", LogCategories.CreateTriggerCategory("Warmup"));
```

Comment:
I had to do this because I had added 
https://github.com/Azure/azure-webjobs-sdk-extensions/blob/515576b4226bcf980792a3a57684e187f0be9d1a/src/WebJobs.Extensions/Extensions/Warmup/WarmupConfigProvider.cs#L34

Guessing it's not a big deal to have this log at every startup. But just wanted to make a note of it in case there's concerns.